covid19-etl/utils/metadata_helper.py

```diff
@@ -92,8 +99,27 @@ def get_latest_submitted_date_idph(self):
         variables = {"filter": {"=": {"project_id": self.project_id}}}
         query_res = self.query_guppy(query_string, variables)
         loc = query_res["data"]["location"][0]
-        latest_submitted_date = datetime.datetime.strptime(loc["date"], "%Y-%m-%d")
-        return latest_submitted_date.date()
+        return loc["date"]
+
+    def get_str_latest_submitted_date_idph_group_demo(self):
+        print("Getting the latest summary_clinical date from Guppy...")
+        query_string = """query ($filter: JSON) {
+            group_demographics (
```

Comment:
i don't see it in the ETL mapping 🤔

Reply:
because the node does not exist. I think it will be another ticket. That's why I have None return value

Reply:
oh i see 🤔 could you create that other ticket then